File: MODUL05/PraktikumC/5.c
#include <stdio.h>

void Biodata(int tahun_lahir, char nama[], char asal[]) {
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang - tahun_lahir;

    printf("Perkenalkan nama saya: %s\n", nama);
    printf("Umur saya: %d tahun\n", umur);
    printf("Saya adalah angkatan: %d\n", tahun_sekarang);
    printf("Asal saya dari: %s\n", asal);
}

int main() {
    int tahunLahir;
    char Namaku[20], Asal[15];

    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c", Namaku);
    scanf(" %[^\n]%*c", Asal);

    Biodata(tahunLahir, Namaku, Asal);

    return 0;
}

File: MODUL05/PraktikumPy/5.py
def Biodata(tahun_lahir, nama, asal):
    tahun_sekarang = 2020
    Umur = tahun_sekarang - tahun_lahir

    print(f"Perkenalkan nama saya: {nama}")
    print(f"Umur: {Umur} tahun")
    print(f"Saya adalah angkatan: {tahun_sekarang}")
    print(f"Asal saya dari: {asal}")

def main():
    tahun_lahir = int(input())
    nama = input()
    asal = input()

    Biodata(tahun_lahir, nama, asal)

if __name__ == "__main__":
    main()